daml-assistant/integration-tests/BUILD.bazel

```diff
@@ -4,26 +4,43 @@ load("//bazel_tools:haskell.bzl", "da_haskell_test")
 
 da_haskell_test(
     name = "integration-tests",
-    timeout = "long",
     srcs = ["src/Main.hs"],
-    data = [
-        "//release:sdk-release-tarball",
-        "@local_jdk//:bin/java",
-    ],
     hazel_deps = [
         "async",
         "base",
         "directory",
         "extra",
         "filepath",
+        "http-client",
+        "http-types",
         "main-tester",
         "network",
         "process",
         "tasty",
         "tasty-hunit",
+        "text",
     ],
     deps = [
         "//daml-assistant/daml-helper:daml-helper-lib",
         "//libs-haskell/bazel-runfiles",
     ],
+    data = [
+        "//release:sdk-release-tarball",
+        "@local_jdk//:bin/java",
+        "@mvn_nix//:bin/mvn",
+        "//:component-version",
+        "//daml-lf/archive:daml_lf_archive_java.jar",
+        "//daml-lf/archive:daml_lf_archive_java_pom.xml",
+        "//language-support/java/bindings:bindings-java",
+        "//language-support/java/bindings:bindings-java_pom.xml",
+        "//language-support/java/bindings-rxjava",
+        "//language-support/java/bindings-rxjava:bindings-rxjava_pom.xml",
+        "//language-support/java/codegen:shaded_binary.jar",
+        "//language-support/java/codegen:shaded_binary_pom.xml",
+        "//ledger-api/rs-grpc-bridge:librs-grpc-bridge.jar",
+        "//ledger-api/rs-grpc-bridge:rs-grpc-bridge_pom.xml",
```

Comment:
I guess this is not trying to be an exhaustive list or artefacts, is it?

Reply:
This is only a list of dependencies that are required by the quickstart example. I verified that it is sufficient by changing the VERSION file to some bogus version that is not yet published to bintray.

Reply:
okay, thank you.